docs/01. installation/07. docker.md

```diff
@@ -153,12 +153,19 @@ You can easily use different credentials by adjusting them in the `docker-compos
 
 It's easy to setup PHPStorm in combination with Docker and Xdebug, one of the most popular tools to debug your PHP application.
 
-- First copy the .env.dist to .env in the root folder of your project
+- First create a file called .env.local in the root folder of your project and paste these lines in them
 ```
-cp .env.dist .env
+# Use this file to set your environment variables for Docker
+
+# XDebug variables
```

Comment:
Docker-compose only looks at .env currently (https://docs.docker.com/compose/environment-variables/#the-env-file). 

I guess we can set the .env.local in the `docker-compose.yml`:

```
    env_file:
     - .env.local
```

To make it point to .env.local, and people can move the xdebug variables there.

Reply:
You can even add multiple env files in the docker-compose env_file directive

Reply:
@jessedobbelaere I hadn't considered people still using Docker for a webserver. I'll add both `.env` and `.env.local`

Reply:
Unfortunately Docker doesn't like loading environment variables from files that don't exist and `.env.local` doesn't always exist so I created a Docker specific `.env` file in `var/docker`